src/analysis/retail/monk/mistweaver/modules/spells/Vivify.tsx

```diff
@@ -44,16 +66,51 @@ class Vivify extends Analyzer {
 
   get suggestionThresholds() {
     return {
-      actual: this.averageRemPerVivify,
+      actual: this.casts > 0 ? this.averageRemPerVivify : 0,
       isLessThan: {
-        minor: 1.75,
-        average: 1.25,
-        major: 0.75,
+        minor: this.casts > 0 ? this.estimatedAverageReMs : 0,
+        average: this.casts > 0 ? this.estimatedAverageReMs - 0.5 : 0,
+        major: this.casts > 0 ? this.estimatedAverageReMs - 1 : 0,
       },
       style: ThresholdStyle.NUMBER,
     };
   }
 
+  get estimatedAverageReMs() {
+    if (this.risingMistActive) {
+      this.expectedAverageReMs = 4.44;
+    } else {
+      this.expectedAverageReMs = 2.22;
```

Comment:
2.22/4.44 are kind of arbitrary and we may want to change them in the future due to talent tuning, so let's put `2.22` into a global const and then use that variable (or variable * 2 if they have RM)